stix_shifter_modules/elastic_ecs/stix_translation/json/to_stix_map.json

```diff
@@ -179,12 +524,146 @@
       "object": "nt",
       "group": "True",
       "transformer": "ToLowercaseArray"
+    },
+    "vlan": {
+      "id": {
+        "key": "x_ecs_network.vlan_id",
+        "cybox": false
+      },
+      "name": {
+        "key": "x_ecs_network.vlan_name",
+        "cybox": false
+      }
+    },
+    "inner": {
+      "vlan": {
+        "id": {
+          "key": "x_ecs_network.inner_vlan_id",
+          "cybox": false
+        },
+        "name": {
+          "key": "x_ecs_network.inner-vlan_name",
```

Comment:
Mistakenly used a hyphen instead of an underscore.

Reply:
fixed